src/com/etendoerp/dependencymanager/DependencyManagerComponentProvider.java

```diff
@@ -27,6 +27,7 @@ public List<ComponentResource> getGlobalComponentResources() {
     // Add all the javascript source files needed in our module
     grhelper.addERP("uninstallDependencyWarnings.js");
     grhelper.addERP("changeVersionDropdown.js");
+    grhelper.addERP("changeFormat.js");
```

Comment:
Consider using an array and a loop for cleaner and more maintainable code:

```java
public static final String[] JS_FILES = {
    "uninstallDependencyWarnings.js",
    "changeVersionDropdown.js",
    "changeFormat.js"
};

for (String file : jsFiles) {
    grhelper.addERP(file);
}
```

This approach simplifies adding multiple files and makes future updates easier